src/Pages/Blog.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Blog = () => {
+  return (
+    <div>
```

Comment:
It would be nice to add className="page-container" to the outermost div so this page has the same style as the other pages